alicloud/service_alicloud_drds.go

```diff
@@ -0,0 +1,350 @@
+package alicloud
+
+import (
+	"fmt"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/drds"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+type DrdsService struct {
+	client *connectivity.AliyunClient
+}
+
+// crate Drdsinstance
+func (s *DrdsService) CreateDrdsInstance(req *drds.CreateDrdsInstanceRequest) (response *drds.CreateDrdsInstanceResponse, err error) {
+
+	//resp, err := s.client.WithDrdsClient(req)
+	raw, err := s.client.WithDrdsClient(func(drdsClient *drds.Client) (interface{}, error) {
+		return drdsClient.CreateDrdsInstance(req)
+	})
+	if err != nil {
+		return nil, fmt.Errorf("create drds error")
+	}
+	resp, _ := raw.(*drds.CreateDrdsInstanceResponse)
+
+	return resp, nil
+}
+
+func (s *DrdsService) DescribeDrdsInstance(drdsInstanceId string) (response *drds.DescribeDrdsInstanceResponse, err error) {
+	req := drds.CreateDescribeDrdsInstanceRequest()
+	req.DrdsInstanceId = drdsInstanceId
+	//resp, err := client.drdsconn.DescribeDrdsInstance(req)
+	raw, err := s.client.WithDrdsClient(func(drdsClient *drds.Client) (interface{}, error) {
+		return drdsClient.DescribeDrdsInstance(req)
+	})
+	if err != nil {
+		return nil, fmt.Errorf("describe drds instance error")
+	}
+	resp, _ := raw.(*drds.DescribeDrdsInstanceResponse)
+
+	if resp == nil {
+		return resp, fmt.Errorf("describe drds instance error")
```

Comment:
If the resp==nil, please build a `NotFoundError` and return

Reply:
fix